src/liballoc/vec.rs

```diff
@@ -324,7 +324,7 @@ impl<T> Vec<T> {
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn new() -> Vec<T> {
```

Comment:
You forgot the actual change of this PR 😆

Reply:
Oh. :man_facepalming: Thanks!